vendor/github.com/mongodb/mongo-tools-common/db/db.go

```diff
@@ -414,3 +415,55 @@ func CanIgnoreError(err error) bool {
 
 	return false
 }
+
+// IsWiredTiger returns whether the storage engine is WiredTiger. Also returns false
+// if the storage engine type cannot be determined for some reason.
+func IsWiredTiger(database *mongo.Database, collectionName string) bool {
```

Comment:
Do we need this function anymore?

Reply:
Not technically, but I thought it could see some use in the future. I'm fine with removing it, if you want.

Reply:
Yeah, I'd rather say "YAGNI" and put it back if there's ever a need.

Reply:
sgtm